package.json

```
@@ -35,8 +36,10 @@
         "react": "18.2.0",
         "react-dom": "18.2.0",
         "react-focus-lock": "2.9.1",
+        "react-i18next": "^12.1.1",
         "react-is": "18.2.0",
         "spatial-navigation-polyfill": "git+https://git@github.com/Stremio/spatial-navigation.git#64871b1422466f5f45d24ebc8bbd315b2ebab6a6",
+        "stremio-translations": "git+https://git@github.com/Stremio/stremio-translations.git#ef047329f5bcb0a8f96008fca05c68b449b34cb1",
```

We can keep translations up-to-date using version to keep track which translations we have in the app, with commit it's not that visible.
- I'll look into releasing versions to `npm` and just use Patch versions and `npm update`.
- ~Could you use a tag version for now?~
```suggestion
        "stremio-translations": "git+https://git@github.com/Stremio/stremio-translations.git#v1.43.16",
```

---

I didn't even know that we started to do tag versioning for stremio-translations
I guess it makes sense to use it if we actually have a regular release cycle, the advantage of using commit hash was that it was quick to update in any app, translations are sparse and doesn't come in bulk most of the time

---

Yes, correct, we will leave it for now to hashes.
There are inherit complications with releasing versions that we will avoid for now.
I've discussed this both with @TheBeastLT and @jaruba